src/stories/Library/search-result-item/SearchResultItem.stories.tsx

```diff
@@ -53,3 +56,11 @@ const Template: ComponentStory<typeof SearchResultItem> = (args) => {
 
 export const Item = Template.bind({});
 Item.args = {};
+
+export const ContentOverload = Template.bind({});
+ContentOverload.args = {
+  title:
+    "Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn Audrey Hepburn",
+  author: "Sánchez Vegara, Amaia Arrazola, Sánchez Vegara, et al.",
+  availabilityLabels: 25,
+};
```

Comment:
:+1: for adding a Story showing a more extreme example. We have actually been provided with a [list of examples which the libraries think demonstrates extremes](https://drive.google.com/file/d/1ox1pXe6YqvNnuGlazFq7uusrfWW11I9f/view?usp=sharing). There is also [one using a very long title](https://www.aakb.dk/ting/object/870970-basis%3A52863767). Consider using that to make the example relatable.